README.md

```diff
@@ -1 +1,93 @@
-# opensearch
\ No newline at end of file
+## Ansible Role: Opensearch
+
+Ansible role to configure wazuh manager standalone with slack integration
+
+Some of the highlighting features are:-
+
+  - Standalone setup of Opensearch
+  - Usermanagement 
+  - 
+ 
+
+### Supported OS
+- Ubuntu 18 
```

Comment:
Should be
  Ubuntu: 
      - focal
      - bionic
  Centos
       - 7
       - 8
  AmaozonLinux:2